### PR TITLE
Document bcntsynctag source parsing support.

### PR DESCRIPTION
**Pull request by adrianschroeter**
Also writing a little bit regarding build counter syncing at all. Still much missing, but it is a start ...

### CODE REVIEW

File: xml/obs_scheduling_and_dispatching.xml [hunk 1]
```diff
@@ -175,4 +175,31 @@
    </variablelist>
   </sect2>
  </sect1>
+ <sect1>
+  <title>Release Number Handling</title>
+  <para>Most build formats define a version and a release number. The Version is usually
```
Comment (hennevogel) [inline xml/obs_scheduling_and_dispatching.xml:180]:
```suggestion
  <para>Most build formats define a version and a release number. The version is usually
```

File: xml/obs_scheduling_and_dispatching.xml [hunk 2]
```diff
@@ -175,4 +175,31 @@
...
+  defined by the up-stream project, while the release number is owned by the distributor.
```
Comment (hennevogel) [inline xml/obs_scheduling_and_dispatching.xml:181]:
```suggestion
  defined by the upstream project, while the release number is handled by the distributor.
```

File: xml/obs_scheduling_and_dispatching.xml [hunk 3]
```diff
@@ -175,4 +175,31 @@
...
+  In addtion to this &obs; differentiates between the a source change (called check-in counter aka CI_CNT)
+  and a build counter (BCNT). The check-in counter gets increased when any source change is done. The build
+  counter gets increased when a build gets trigger due to any other reason than a source change.</para>
+  <para>The version and check-in counter are defined by the package source, while the build counter is
+  stored per project, package, repository and architecture.
```
Comment (hennevogel) [inline xml/obs_scheduling_and_dispatching.xml:186]:
```suggestion
  stored per project, package, repository and architecture combination.
```

File: xml/obs_scheduling_and_dispatching.xml [hunk 4]
```diff
@@ -175,4 +175,31 @@
...
+   <para>The default configuration is to sync build counters across all architectures for a given
+   project, package and repository. This can only be changed by the OBS administrator.
+   </para>
+  </sect2>
+  <sect2>
+   <title>Build Counter Syncing via multiple packages</title>
+   <para>In some cases it is critical keep the build counter in sync via multiple packages.
+   The old way to do so is to define a common bcntsynctag in package meta. The used string
+   identifies the packages to be kept in sync. The package name is the default when no
+   bcntsynctag exists. Please note that this affects always all flavors of a multibuild source.
```
Comment (hennevogel) [inline xml/obs_scheduling_and_dispatching.xml:199]: I think you should link to the examples how to set both.

**Review by hennevogel — CHANGES_REQUESTED** [review]

File: xml/obs_scheduling_and_dispatching.xml [hunk 5]
```diff
@@ -175,4 +175,31 @@
    </variablelist>
   </sect2>
  </sect1>
+ <sect1>
+  <title>Release Number Handling</title>
+  <para>Most build formats define a version and a release number. The Version is usually
+  defined by the up-stream project, while the release number is owned by the distributor.
+  In addtion to this &obs; differentiates between the a source change (called check-in counter aka CI_CNT)
+  and a build counter (BCNT). The check-in counter gets increased when any source change is done. The build
+  counter gets increased when a build gets trigger due to any other reason than a source change.</para>
```
Comment (tomschr) [inline xml/obs_scheduling_and_dispatching.xml:184]:
```suggestion
  counter gets increased when a build gets triggered due to any other reason than a source change.</para>
```

**Review by tomschr — CHANGES_REQUESTED** [review]
Minor typo

File: xml/obs_scheduling_and_dispatching.xml (hunk 4 above)
Comment (adrianschroeter) [reply xml/obs_scheduling_and_dispatching.xml:199]: well, there is much missing here, but no time atm for this.

**Pull request merged**